RetailCoder.VBE/Inspections/QuickFixes/AssignedByValParameterQuickFix.cs

```diff
@@ -70,75 +75,46 @@ private void SetLocalCopyVariableName()
             }
         }
 
-        private void ModifyBlockToUseLocalCopyVariable()
+        private bool ProposedLocalVariableNameIsValid()
```

Comment:
There is already a VariableNameValidator class (used internally in this QuickFix as well) that has the generic checks for variable name validity - only to the scope of the variable name as a standalone name.  This function would resist extraction a bit in that it has an additional check beyond the VariableNameValidator class (e.g., the variable name is already used somewhere in the procedure).